docs/builtin_elements.md

```diff
@@ -890,6 +892,17 @@ Depending on the backend and used OS unidirectional resize cursors may be replac
 * **`nesw-resize`**: Bidirectional resize north-east-south-west.
 * **`nwse-resize`**: Bidirectional resize north-west-south-east.
 
+## `InputType`
+
+This enum is used to define the type of the input field. Currently this only differentiates between
+text and password inputs but in the future it could be expanded to also define what virtual keyboard
+should be shown
```

Comment:
```suggestion
text and password inputs but in the future it could be expanded to also define what type of virtual keyboard
should be shown, for example.
```